src/pages-and-resources/discussions/app-config-form/AppConfigForm.jsx

```diff
@@ -78,12 +78,21 @@ function AppConfigForm({
     alert = <PermissionDeniedAlert />;
   }
 
-  let form = null;
+  let form;
   if (selectedAppId === 'legacy') {
     form = (
-      <LegacyConfigForm
+      <OpenedXConfigForm
         formRef={formRef}
         onSubmit={handleSubmit}
+        legacy
+      />
+    );
+  } else if (selectedAppId === 'openedx') {
+    form = (
+      <OpenedXConfigForm
+        formRef={formRef}
+        onSubmit={handleSubmit}
+        legacy={false}
       />
```

Comment:
I think this code can be shortened this way

`<OpenedXConfigForm
        formRef={formRef}
        onSubmit={handleSubmit}
        legacy={selectedAppId !== 'openedx'}
      />`

Reply:
@xitij2000 can you please rebase this branch so I can I have a final review on it. Thanks.

Reply:
I'm not sure how that shortening will work? There are three conditions, either it's opened, or legacy or something else. The above will match for legacy and other lti providers which we don't want.